Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml

```diff
@@ -241,20 +241,216 @@ tasks:
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      asm_id_list:
+        complex:
+          accessor: xpanseassetids
+          root: incident
+          transformers:
+          - args:
+              chars:
+                value:
+                  simple: '[]'
+            operator: StripChars
+          - args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: '"'
+            operator: replace
+      tags:
+        complex:
+          accessor: Answers}
+          root: ${Determine tags to add to asset
+          transformers:
+          - operator: LastArrayElement
+          - args:
+              field:
+                value:
+                  simple: "0"
+            operator: getField
+          - args:
+              delimiter:
+                value:
+                  simple: ','
+            operator: split
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Cortex Xpanse
+      description: Assigns tags to a list of assets.
+      id: 3279b4b2-f6ca-4f10-8bcf-c199409bcf5f
+      iscommand: true
+      name: Assign Tag to Asset(s)
+      script: Cortex Xpanse|||asm-tag-asset-assign
+      type: regular
+      version: -1
+    taskid: 3279b4b2-f6ca-4f10-8bcf-c199409bcf5f
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1180
+        }
+      }
+  "10":
+    conditions:
+    - condition:
+      - - ignorecase: true
+          left:
+            iscontext: true
+            value:
+              simple: inputs.WriteToXpanse
+          operator: isEqualString
+          right:
+            value:
+              simple: "True"
+      label: "yes"
+    continueonerrortype: ""
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#default#':
+      - "2"
+      "yes":
+      - "7"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Determine whether the playbook should write data back to Cortex Xpanse.
+      id: 62fb488b-6b5a-469d-800e-80a6e253d8b1
+      iscommand: false
+      name: Are we writing to Cortex Xpanse?
+      type: condition
+      version: -1
+    taskid: 62fb488b-6b5a-469d-800e-80a6e253d8b1
+    timertriggers: []
+    type: condition
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 790
+        }
+      }
 view: |-
   {
-    "linkLabelsPosition": {},
+    "linkLabelsPosition": {
+      "10_7_yes": 0.49
+    },
     "paper": {
       "dimensions": {
-        "height": 1125,
+        "height": 1515,
         "width": 380,
         "x": 450,
         "y": -80
       }
     }
   }
-inputs: []
+inputs:
+- description: Whether to write data back to Cortex Xpanse. 
```

Comment:
Based on the RN, analysts should also add tags to assets using this input. Can we add this sentence also to this description?

Reply:
This is only to determine if data is written back to Xpanse.  Currently it is only for tagging, but could be other things in the future.  I added this so that the default behavior is to bypass the data collection task.  Please let me know if I should add something, but I think it is good as-is.

Reply:
up to you.
And yet, since you will request certain data from the analyst and the XSOAR user who is configuring this input needs to know why it's there - I suggest adding at least a sentence for each component it will require.

Such as:
Data to be written back via data collection from the user:
- Tags

And the rest should be in the future

Reply:
I would prefer to keep it generic